docs/specs/2026-05-03-backlog-map/admin-panel-top10.md

```
@@ -0,0 +1,102 @@
+**Status**: Pending
+`DEVELOPER_USER_IDS=282294772570521600` written to `.env` but container not restarted. All admin routes return 403 for the developer until this runs.
+```
+docker compose restart lucky-backend
+```
+
+---
+
+### 2. Run DB migration for `global_feature_toggles` [DEPLOY] — S
+**Status**: Pending
+`prisma/migrations/20260504000000_add_global_feature_toggles/migration.sql` exists but not applied on homelab. Toggle writes will throw at runtime.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)